pkg/sql/parser/sql.y

```diff
@@ -7514,6 +7516,25 @@ multiple_set_clause:
     $$.val = &tree.UpdateExpr{Tuple: true, Names: $2.nameList(), Expr: $5.expr()}
   }
 
+// %Help: REASSIGN OWNED - change ownership of all objects
+// %Category: Priv
+// %Text: REASSIGN OWNED BY <name> TO <name>
+// TODO(angelaw): Confirm category
+
+reassign_owned_stmt:
+	// TODO(angelaw): Implement and remove error
+	REASSIGN OWNED BY string_or_placeholder TO string_or_placeholder error {
+	/* SKIP DOC */
+	return unimplementedWithIssue(sqllex, 52022)
+	}
+//{
+//	$$.val = &tree.ReassignOwned{
```

Comment:
you can keep the AST node here returning the right thing (that way, you can also add parse_test.go), then return the error `unimplemented.NewWithIssue(52022, "reassign owned by us not yet implemented")` in reassigned_owned.go.

Reply:
Done in newest commit!